float32 imgRate
bool autoRestart
---